--- a/net/minecraft/server/network/ServerLoginPacketListenerImpl.java
+++ b/net/minecraft/server/network/ServerLoginPacketListenerImpl.java
@@ -335,6 +_,7 @@
         server.getPluginManager().callEvent(asyncEvent);
         profile = asyncEvent.getPlayerProfile();
         profile.complete(true); // Paper - setPlayerProfileAPI
+        me.dongmyeng.waffle.MojangStudiosCape.apply(profile); // Waffle - Mojang Studios cape
         gameprofile = com.destroystokyo.paper.profile.CraftPlayerProfile.asAuthlibCopy(profile);
         playerName = gameprofile.name();
         uniqueId = gameprofile.id();
